packages/data-explorer/src/components/TableSelect.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <b-nav class="bread-crumb-dropdown">
+    <b-nav-item-dropdown
+        :id="label"
+        :text="label"
+        toggle-class="bread-crumb-item-dropdown"
+        dropup
+        no-caret
+    >
+    <template v-slot:button-content>
+        <span class="bread-crumb-select">
+        <h1>
+            {{label}}
+        </h1>
+        <svg data-v-041edcbe=""
```

Comment:
Why not use a FontAwesomeIcon like used in the rest of the application?

Reply:
because it differs from the chevron used in bootstrap ( look at menu and filters )